Muka-is-home/api
# added cloudinary upload to blog and profile image

This PR adds Cloudinary upload functionality to the blog forms as well as user profile forms. 

To test:
- ```shell 
  python manage.py migrate
  ```
- ```shell
  python manage.py runserver
  ```
-  Log in as a superuser to test blog create and edit functionality
- Log in as a user to test profile create and edit functionality

AC:
-  A superuser can upload images to Cloudinary in the shop and blog forms
-  A standard user can upload images to Cloudinary in the create and edit user profile forms.
- When editing an image in any form, the new photo should display upon submit.
